hackathon_site/dashboard/frontend/src/slices/order/adminOrderSlice.ts

```diff
@@ -46,7 +46,7 @@ interface RejectValue {
 
 export const getOrdersWithFilters = createAsyncThunk<
     APIListResponse<Order>,
-    void,
+    { keepOld?: boolean } | undefined,
```

Comment:
we don't need keepOld in this case, that was added to the hardware thunk because it required more flexibility. So you can remove this.